tests/ResourceSyncingTest.php

```diff
@@ -149,7 +150,7 @@
     });
 
     // When central model provides the list of attributes, resource model will be created from the provided list of attributes' values
-    $centralUser->tenants()->attach('t1');
+    $centralUser->resources()->attach('t1');
```

Comment:
The central user *is the resource*. Why would we attach the tenant to `resources()`? Is the name change needed?

Reply:
Yes, it doesn't make sense. I renamed it to `tenants`.